### PR TITLE
307 dashboard improvements

### LINKED ISSUE #307: Improve "dashboard"
## Acceptance Critera

- None. Just add more information and functionality to it. Right now it doesn't do much. Anything we add is an improvement.

### PR DESCRIPTION
**Pull request by jimmykarily**
Fixes #307 

This is a first iteration. Next step would be to allow:

- creation of orgs
- listing available services and plans
- creation of services (custom or other)
- binding/unbinding services

from the web ui

### CODE REVIEW

**Comment by jimmykarily** [comment]
This is what it currently looks like:

image

Changes:

- Moved the org specific links to the left navigation bar
- Created a drop down that changes the current org (as in `epinio target`). It uses a cookie to store the current org.
- Added favicons
- Added logo
- Formatted the urls. They are now clickable links
- Added icon next to the "Application" link

**Comment by jimmykarily** [comment]
Added a modal to create organizations:

image

**Comment by jimmykarily** [comment]
Icons from here: https://feathericons.com/

File: internal/web/applications_controller.go [hunk 1]
```diff
@@ -6,15 +6,106 @@ import (
...
 
 type ApplicationsController struct {
 }
 
+func setCurrentOrgInCookie(org, cookieName string, w http.ResponseWriter) error {
+	cookie := http.Cookie{
+		Name:    cookieName,
+		Value:   org,
+		Path:    "/",
+		Expires: time.Now().Add(365 * 24 * time.Hour),
```
Comment (andreas-kupries) [inline internal/web/applications_controller.go:23]: Are there `time.Day`, or even `time.Year` which could be used here to simplify things ?

**Review by andreas-kupries — DISMISSED** [review]
The parts I understood (mostly the go code) look ok.
I can't be as sure about .js parts.

File: internal/web/applications_controller.go (hunk 1 above)
Comment (andreas-kupries) [reply internal/web/applications_controller.go:23]: Retracting. Cannot find anything larger than `duration.Hour`.

**Pull request merged**